src/main/kotlin/org/ozwillo/dcimporter/service/rabbitMQ/MarcheSecuriseReceiver.kt

```diff
@@ -132,9 +136,16 @@ class MarcheSecuriseReceiver (val marcheSecuriseService: MarcheSecuriseService,
                             logger.warn(e.message)
                         }
                     }
+
+                    routingBindingKeyOfType(routingKey, MSUtils.REPONSE_TYPE) -> channel.basicAck(tag, false)
```

Comment:
can't you just ignore them and let them in the `else` below ?

Reply:
Doing so will trigger a warn log whereas everything is ok. We let them here for now, it still can be improved later.